lib/question.js

```diff
@@ -1772,17 +1772,18 @@ module.exports = {
   _buildGradingJobStats(job) {
     if (job) {
       const phases = [];
-      const totalDuration = moment
-        .duration(moment(job.grading_requested_at).diff(job.graded_at))
-        .asMilliseconds();
+      const totalDuration = differenceInMilliseconds(
+        parseISO(job.graded_at),
+        parseISO(job.grading_requested_at)
+      );
       const formatDiff = (start, end, addToPhases = true) => {
-        const duration = moment.duration(moment(end).diff(start)).asMilliseconds();
+        const duration = differenceInMilliseconds(parseISO(end), parseISO(start));
         if (addToPhases) {
-          const percentage = -1 * (duration / totalDuration);
+          const percentage = duration / totalDuration;
           // Round down to avoid width being greater than 100% with floating point errors
           phases.push(Math.floor(percentage * 1000) / 10);
         }
-        return moment.utc(duration).format('s.SSS');
```

Comment:
I believe this was actually a bug. If the duration was longer than a minute, this would return only the seconds component of the time, not the minutes.

Reply:
Nice catch!